src/components/group-management/member-management-dialog/styles.scss

```diff
@@ -1,7 +1,7 @@
 @use '~@zero-tech/zui/styles/theme' as theme;
 
 .remove-member-dialog {
-  width: 586px;
+  width: 600px;
```

Comment:
Why keep increasing this? Once the dialog has a size it should pretty much stay that way and whichever text has to wrap around will just wrap around.

Reply:
it is related to this PR, [https://github.com/zer0-os/zOS/pull/1920](https://github.com/zer0-os/zOS/pull/1920]. I hadn't increased it properly so there was still some space at the right of modal, but because of `586px` width the text would wrap around it and it would look weird.

Reply:
What is providing the actual full size of the modal? Some other place?... if so, don't you just want this text to fill its width? Why hard code a number here?